salmon/tagger/sources/itunes.py

```diff
@@ -10,37 +12,36 @@
 ALIAS_GENRE = {
     "Hip-Hop/Rap": {"Hip Hop", "Rap"},
     "R&B/Soul": {"Rhythm & Blues", "Soul"},
+    "Music": {},
```

Comment:
Would it be better to handle the edge-case when no genres are indicated by making it at least adding the decade one instead of "music" and keep it only to the tracker upload form? I'm assuming that's where a problem like this would arise

Reply:
This line replaces the genre 'music' with nothing (an empty set). Because apple music now says all music is the genre music. We don't want to tag our music 'Music'. Smoked-Salmon already forces you to pick a genre if it doesn't find one.

Reply:
Oh I was assuming it was the other way around, my bad.